### PR TITLE
USB: ASCII Trance Vib emulation

### PR DESCRIPTION
**Pull request by Florin9doi**
### Description of Changes
Add ASCII Trance Vib emulation

### Suggested Testing Steps
Check the following games:
| Game | Supported |
| --- | --- |
| Rez | ✅ |
| Space Channel 5 Part 2 | ✅ |
| Zettai Zetsumei Toshi | ✅ |
| Disaster Report | ❌ |
| SOS: The Final Escape | ❌ |

### CODE REVIEW

File: pcsx2-qt/resources/icons/black/svg/trance-vibrator-line.png
Comment (JordanTheToaster) [inline pcsx2-qt/resources/icons/black/svg/trance-vibrator-line.png:1]: Can you make these icons SVG?

**Review by JordanTheToaster — CHANGES_REQUESTED** [review]

**Comment by JordanTheToaster** [comment]
Also if you could write a proper PR description that would be good.

File: pcsx2-qt/resources/icons/black/svg/trance-vibrator-line.png
Comment (Florin9doi) [reply pcsx2-qt/resources/icons/black/svg/trance-vibrator-line.png:1]: done

**Review by F0bes — APPROVED** [review]
LGTM. Tested Rez and it detects and uses the device. The vibration worked fine on my controller.

**Pull request merged**